docs/integrations/custom-operations.md

```diff
@@ -0,0 +1,25 @@
+# Custom API operations
+
+n8n supplies hundreds of nodes, allowing you to create workflows that link multiple products. However, some nodes don't include all the possible operations supported by a product's API. You can work around this by making a custom API call.
+
+One of the most complex parts of setting up API calls is credentials. To simplify this, n8n provides a way to use the [HTTP Request](/integrations/core-nodes/n8n-nodes-base.httpRequest/) node with either an existing credential, or using n8n's credential creation process. This avoids complex credentials setups. Creating an HTTP request can require knowledge of different authentication types, and work to set up the appropriate option (such as basic auth, header, or OAuth2.). n8n avoids this complexity using app-specific HTTP credentials.
+
+## App-specific HTTP credentials
```

Comment:
All credentials are app-specific, including if you create them yourself using a generic credential type. And as mentioned yesterday, I'm not sure what 'HTTP' brings.

Why not talk about 'Predefined credential types'?